### PR TITLE
Create the landing page

### PR DESCRIPTION
**Pull request by alexyorke**
This PR creates the landing page. The logo is licensed as "Free for commercial use No attribution required" and is available at https://pixabay.com/photos/planets-orbit-space-sky-world-4584048/

The `Read More` links on the homepage go to "#TBD" as there aren't any pages for them, except for "Flexible Deployment" as it goes to the installation page.

This is linked to issue 1888 internally.

### CODE REVIEW

File: Orbital-Docs/_config.yml [hunk 1]
```diff
@@ -18,30 +18,30 @@
...
 email: opensource@focisolutions.com
 description: # this means to ignore newlines until "baseurl:"
-baseurl: '' # the subpath of your site, e.g. /blog
-url: '' # the base hostname & protocol for your site, e.g. http://example.com
-twitter_username: jekyllrb
+baseurl: "" # the subpath of your site, e.g. /blog
+url: "" # the base hostname & protocol for your site, e.g. http://example.com
+title: ""
 github_username: FociSolutions
-minimal_mistakes_skin: 'contrast'
```
Comment (mariareyesfoci) [inline Orbital-Docs/_config.yml]: I think your prettier is changing from double quotes to single. Can we change this to single?

**Review by JohnnyFoci — APPROVED** [review]
I think we need to have a meeting to decide how all this information will be formatted for the the landing page but it looks great getting content onto the landing page.

File: Orbital-Docs/_config.yml (hunk 1 above)
Comment (alexyorke) [reply Orbital-Docs/_config.yml]: Changed!

**Review by mariareyesfoci — APPROVED** [review]

File: Orbital-Docs/_config.yml [hunk 2]
```diff
@@ -18,30 +18,30 @@
...
-    - label: 'GitHub'
+    - label: "GitHub"
       icon: fab fa-fw fa-github
-      url: 'https://github.com/FociSolutions/Orbital'
-    - label: 'Contribute'
-      url: '/docs/contribute'
-    - label: 'Contact Us'
-      url: 'https://github.com/FociSolutions/Orbital'
+      url: "https://github.com/FociSolutions/Orbital"
+    - label: "Contribute"
```
Comment (mariareyesfoci) [inline Orbital-Docs/_config.yml]: missing to return the single quotes in the rest of the document

Comment (alexyorke) [reply Orbital-Docs/_config.yml]: Fixed

**Pull request merged**